### ISSUE TITLE
is this active?

### ISSUE THREAD

**Issue by stellanhaglund** [old-backside-byu-project/backside#2]

**Comment by trevordixon** [comment]
Nope. It was never quite finished, and nobody's working on it anymore.

**Comment by addisonj** [comment]
I hope to get back to it sometime... but if you want to contribute, please
do!

Addison

**Comment by stellanhaglund** [comment]
I would like to do that, not sure how much i can assist you with though,
but would you mind giving me a small overview on the tools needed and what
parts are missing?

lördag 29 augusti 2015 skrev Addison Higham notifications@github.com:

**Comment by stellanhaglund** [comment]
Is there much parts missing?

lördag 29 augusti 2015 skrev Stellan Haglund sh@idly.se:

**Comment by stellanhaglund** [comment]
I have given it some thought and im very interested in making a project like this to work, would it be better to start from scratch with a new project with its own license or what do you think?

**Comment by stellanhaglund** [comment]
Sorry, me AGAIN.. :)
I hope this isnt a rude question, but would it be possible that i can take
over the project and try to move it forward?
And if that is not an option could I use what you have done to make another
project?

Im just asking this because it seems like a really interesting project, and
since there has been no activity in such a long time then maybe I would be
able to take a bit further..

lördag 29 augusti 2015 skrev Stellan Haglund sh@idly.se:

**Comment by addisonj** [comment]
Hey, no worries... sorry for the delay in getting back to you!

So to be a bit more concrete on the state of this project, it is mostly working on the backend side, with a few larger-ish bugs/changes away from cutting an initial beta release. You can see most of those as issue in the individual projects, for example: https://github.com/backside/backside-api/issues/6 and https://github.com/backside/backside-mongo-store/issues/1 are the two biggest issues on the persistence layer. With those fixed, the persistence layer and API are in a decent shape with most of the core firebase API being supported, but with probably a few unknown bugs.

With that work, the biggest remaining chunk of work is on the client side library. We have some basic stuff done (here: https://github.com/backside/backside-client), but its really the barebones. And probably has lots of bugs.

As far as how to involve you, if you are interested, it would be great if you wanted to help flesh out the functionality and get things in a better state and have you become a maintainer. However, I put a lot of time into this last year and often think about getting back to work on it and am still interested myself in continuing on with it (and have plans for it being the backing store for a number of other projects I am interested in) but real life has been a problem. So rather than just drop it all in your lap, how does having a google hangout or something and going over the architecture and deciding for a way to get your feet wet sound? If that all works well, we can make you a maintainer on it and hopefully also serve as the momentum I need to get back into it.

If that sounds like a decent plan, let me know and we can set something up. If meeting sounds like too much effort, feel free to just fork and hack away and I will review PRs, give feedback and whatever else you need to keep pushing forward.

Glad to see your interest! Hope we can figure something out!

**Comment by addisonj** [comment]
Oh also, as far as license, if we didn't add one, just assume MIT/ISC.

**Comment by stellanhaglund** [comment]
I would be happy to do a hangout, so should we move this conversation to
gmail first maybe?

måndag 31 augusti 2015 skrev Addison Higham notifications@github.com:

**Comment by stellanhaglund** [comment]
I saw an email in one of the package files, don't know if it was the right one..
